ee/clickhouse/views/actions.py

```diff
@@ -96,7 +96,7 @@ def people(self, request: Request, *args: Any, **kwargs: Any) -> Response:
 
         return Response(
             {
-                "results": [{"people": serialized_people[0:100], "count": len(serialized_people[0:99])}],
+                "results": [{"people": serialized_people[0:100], "count": len(serialized_people[0:100])}],
```

Comment:
is there a reason we're returning a count of 99 vs the 100 people that are actually returned? 🤔 @fuziontech

Reply:
I think this is or was used somewhere as a lazy "there are more" indicator. 🤷 

I'd remove it if it doesn't make sense and you can't find any place that uses this to use this. :)

Reply:
Actually, there's a better fix. First, use a `limit` param in the backend. Then modify the backend query to always limit to `n + 1` rows. Then return the first `n` and also a `"has_more": True` if you actually got `n + 1`. That way you can confidently show "found 100" or "found 100+" results.

Reply:
oh yeah 100+ makes sense! I can also just use the "next_url" key for that, I think that returns a string when there are more results~